File: Pong with Brackeys/Assets/Scripts/Menu/MainMenu.js
#pragma strict

import UnityEngine.SceneManagement;

public function Play2Player() {
    SceneManager.LoadScene(1);
}

public function PlaySinglePlayer(){
	SceneManager.LoadScene(2);
}

public function PlaySinglehardPlayer(){
	SceneManager.LoadScene(3);
}
public function QuitGame() {
    Debug.Log("Quit");
    Application.Quit();
}

File: Pong with Brackeys/Assets/Scripts/Menu/MainMenu.cs
using UnityEngine;
using System.Collections;
using UnityEngine.SceneManagement;

public class MainMenu : MonoBehaviour {

	public void PlaySinglePlayer () {
		SceneManager.LoadScene (2);
	}

	public void Play2Player (){
		SceneManager.LoadScene (1);
	}
	public void QuitGame(){
		Debug.Log ("Quit");
		Application.Quit ();
	}
}
